radio-packet-format/sections/error.tex

```diff
@@ -0,0 +1,52 @@
+\subsection{Error Message}
+
+The error message block type reports information about failures in the telemetry system. The format of the error
+message payload is described in figure \ref{format:error-message}.
+
+\begin{figure}[H]
+    \centering
+    \begin{bytefield}[bitwidth=0.03\linewidth]{16}
+        \bitheader{0-15} \\
+        \wordbox{1}{Measurement Time} \\
+        \bitbox{5}{Originating Process} \\
+        \bitbox{3}{Error Level} \\
+        \bitbox{8}{Error Code} \\
+    \end{bytefield}
+    \caption{Error message data block format}
+    \label{format:error-message}
+\end{figure}
+
+\blocktimestampexp
+
+\paragraph{Originating Process}
+
+The originating process is a number that corresponds to the context that the error occured in, denoting which process, thread, or
+hardware component the error occured in. A value of 0x00 represents a general error, not specific to any context. Other values
+are left to the telemetry system to define.
+
+\paragraph{Error Level}
+
+The error level describes the severity of an error. The possible values of this field are described in \ref{table:error-levels}.
+
+\begin{table}[H]
+    \centering
+    \begin{tabular}{@{}ll@{}}
+        \toprule
+        Error level             & Value             \\
+        \midrule
+        Debug                   & 0x00              \\
+        Info                    & 0x01              \\
+        Warning                 & 0x02              \\
+        Error                   & 0x03              \\
+        Fatal                   & 0x04              \\
```

Comment:
I could maybe see an error packet with the process ID, but I don't really understand the utility of anything else. What are we going to do if we receive an error block that says "INFO: Process 2"? That doesn't really tell us anything. Anything below a critical error probably should not be transmitted in flight.